src/memory.c

```diff
@@ -95,6 +139,14 @@ real_read6502(uint16_t address, bool debugOn, uint8_t bank)
 void
 write6502(uint16_t address, uint8_t value)
 {
+	//Update RAM access flag
+	if (address < 0xa000) {
```

Comment:
better guard this with `if (reportUninitializedAccess)`; I fear a speed impact otherwise